packages/web-app-files/src/views/spaces/GenericSpace.vue

```diff
@@ -664,11 +670,16 @@ export default defineComponent({
 
       const debounced = debounce(({ unobserve }) => {
         unobserve()
+        const processor =
+          this.viewMode === ViewModeConstants.tilesView.name
+            ? ProcessorType.enum.fit
+            : ProcessorType.enum.thumbnail
```

Comment:
would it make sense to add a `.processorType` prop to the view mode constants?

Reply:
we quickly discussed it and agreed, we don't want to do it right now. 

I will probably touch this code soon to make it possible to define viewModes as extensions, will revisit it then.